### PR TITLE
add notes on the current status of the repo

### PR DESCRIPTION
**Pull request by isegall-da**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Reference applications for funding, operating, and incentivizing the use of a de
...
+Splice introduces a reference method for operating a publicly available decentralized Canton synchronizer. Each node in the decentralized synchronizer is operated by an entity known in Splice as a "Super Validator". Splice refers to a group of Super Validators actively operating nodes in a decentralized synchronizer at any point in time as the "decentralized synchronizer operator" (dso). The Splice code uses a code construct called a "decentralized synchronizer operator party" (dso party) to accumulate signatures from and take actions on behalf of the currently-active set of Super Validators.
```
Comment (waynecollier-da) [inline README.md:13]: Better to remove the entire sentence:

Splice refers to a group of Super Validators actively operating nodes in a decentralized synchronizer at any point in time as the "decentralized synchronizer operator" (dso).

Comment (waynecollier-da) [reply README.md:13]: The following sentence is fine. We just removed all concepts of a group; no collective noun for SVs. We just refer to the SVs as the SVs.

File: README.md [hunk 2]
```diff
@@ -113,4 +113,23 @@ The following diagram shows how the Splice applications interact with the Daml p
...
+The SV App module contains the Amulet smart contract code and the Amulet configuration variables, as well as the Synchronizer Governance app.
+
+## Repository Status
+
+This repository is in the process of being contributed from the private repository of Digital Asset.
+
+As of now, it has the following limitations:
+
+- No CI or other periodic testing is running directly on this repository. We update this
+  repository daily as a dump of a private repository at Digital Asset, which is currently
```
Comment (waynecollier-da) [inline README.md]: OSS repo?

**Review by waynecollier-da — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (isegall-da) [reply README.md:13]: Note that there's no change here, this is just my editor deleting the extra whitespace at the end of the line.
Let's update this in a separate PR, to keep them atomic meaningful updates.

**Pull request merged**